en/08_Changelogs/5.4.0.md

```diff
@@ -222,7 +222,7 @@ The `SilverStripe\CMS\Model\SiteTree.DESCRIPTION` localisation key (along with t
 - The [`SiteTree.icon`](api:SilverStripe\CMS\Model\SiteTree->icon) configuration property has been deprecated. It will be renamed to `cms_icon`.
 - The [`SiteTree.icon_class`](api:SilverStripe\CMS\Model\SiteTree->icon_class) configuration property has been deprecated. It will be renamed to `cms_icon_class`. This applies to all subclasses of `SiteTree` as well.
 - [`CMSMain::PageList()`](api:SilverStripe\CMS\Controllers\CMSMain::PageList()) has been deprecated. It will be renamed to `RecordList()`.
-- [`CMSMain::PageListSidebar()`](api:SilverStripe\CMS\Controllers\CMSMain::PageListSidebar()) has been deprecated. It will be removed without equivalent functionality.
+- [`CMSMain::PageListSidebar()`](api:SilverStripe\CMS\Controllers\CMSMain::PageListSidebar()) has been deprecated. It will be removed without equivalent functionality to replace it.
```

Comment:
Tidying this up while I'm here to be consistent with the other deprecation messages in this list